modules/signatures/windows/loader_hancitor.py

```diff
@@ -0,0 +1,64 @@
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import base64
+import re
+
+from lib.cuckoo.common.abstracts import Signature
+
+class Hancitor(Signature):
+    name = "loader_hancitor"
+    description = "Creates known Hancitor loader URLs and/or C2 request"
+    severity = 3
+    categories = ["loader"]
+    families = ["hancitor"]
+    authors = ["ronbarrey"]
+    minimum = "2.0"
+
+    urls_re = [
+        ".*api.ipify.org",
+        ".*com\/ls[0-9]\/.*.php",
+        ".*ru\/ls[0-9]\/.*.php",
```

Comment:
probably better remove tlds as it can be changed easy as path, but it will be better in my opinion